build.fsx

```diff
@@ -164,7 +164,7 @@ Target "NuGet" <| fun () ->
 // Generate the documentation
 
 Target "GenerateDocs" <| fun () ->
-    executeFSIWithArgs "docs/tools" "generate.fsx" ["--define:RELEASE"] []
+    executeFSIWithArgs "docs/output" "generate.fsx" ["--define:RELEASE"] []
```

Comment:
ok. It didn't work because directory didn't exist, and most matching path between other occurences was docs/output, so i just changed it without thinking, will revert for now